notebook/static/tree/js/newnotebook.js

```diff
@@ -63,6 +63,7 @@ define([
                 .attr("id", "kernel-" +ks.name)
                 .data('kernelspec', ks).append(
                     $('<a>')
+                        .attr("aria-label","list item 1,"+ks.name)
```

Comment:
Will this not result in multiple `list item 1`?

Reply:
This has been made dynamic